Install perl module in external functional dockerfile failed.

https://ci.adoptopenjdk.net/view/Test_external/job/openjdk8_j9_externaltest_extended_x86-64_linux/169/consoleFull

```
07:10:55       [exec] [91mExpat.xs:12:10: fatal error: expat.h: No such file or directory
07:10:55       [exec]  #include <expat.h>
07:10:55       [exec]           ^~~~~~~~~
07:10:55       [exec] compilation terminated.
07:10:55       [exec] [0m[91mmake[1]: *** [Expat.o] Error 1
07:10:55       [exec] [0mMakefile:319: recipe for target 'Expat.o' failed
07:10:55       [exec] make[1]: Leaving directory '/root/.cpan/build/XML-Parser-2.44-0/Expat'
07:10:55       [exec] [91mmake: *** [subdirs] Error 2
07:10:55       [exec] [0mMakefile:538: recipe for target 'subdirs' failed
07:10:55       [exec]   TODDR/XML-Parser-2.44.tar.gz
07:10:55       [exec]   /usr/bin/make -- NOT OK

```
Looks like there are some lib dependent.  Will investigate.

--- a/thirdparty_containers/functional-test/dockerfile/Dockerfile
+++ b/thirdparty_containers/functional-test/dockerfile/Dockerfile
@@ -46,10 +46,8 @@ RUN apt-get update \
 
 RUN apt-get -y install \
 	libtext-csv-perl \
-	libjson-perl
-
-# Install Docker module to run test framework
-RUN echo yes | cpan install JSON Text::CSV XML::Parser
+	libjson-perl \
+	libxml-parser-perl
 
 # Install ant
 ENV ANT_VERSION 1.10.5

--- a/thirdparty_containers/system-test/dockerfile/Dockerfile
+++ b/thirdparty_containers/system-test/dockerfile/Dockerfile
@@ -63,7 +63,10 @@ VOLUME ["/java"]
 ENV  JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
 
 # Install Docker module to run test framework
-RUN echo yes | cpan install JSON Text::CSV XML::Parser
+RUN apt-get -y install \
+	libtext-csv-perl \
+	libjson-perl \
+	libxml-parser-perl
 
 # This is the main script to run system tests.  
 COPY ./dockerfile/system-test.sh /system-test.sh

